qt/src/SourceManager.hh

```diff
@@ -24,13 +24,18 @@
 #include <QStringList>
 #include <QFileSystemWatcher>
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+#include <poppler-qt4.h>
+#else
+#include <poppler-qt5.h>
+#endif
+
```

Comment:
Forward-declare instead of including header

Reply:
Fixed.